hal/armv7m/imxrt/117x/imxrt.h

```diff
@@ -38,6 +37,32 @@ enum { pctl_clk_cm7 = 0, pctl_clk_cm4, pctl_clk_bus, pctl_clk_bus_lpsr, pctl_clk
 	pctl_clk_mipi_esc, pctl_clk_csi2, pctl_clk_csi2_esc, pctl_clk_csi2_ui, pctl_clk_csi, pctl_clk_ccm_clko1,
 	pctl_clk_ccm_clko2 };
 
+
+/* CCM - Low Power Clock Gates */
+enum { pctl_lpcg_m7, pctl_lpcg_m4, pctl_lpcg_sim_m7, pctl_lpcg_sim_m, pctl_lpcg_sim_disp, pctl_lpcg_sim_per,
```

Comment:
First element of the `enum` should be assigned to `0`.

Reply:
Ok, resolved.